perceval/utils/conversion.py

```diff
@@ -59,13 +59,18 @@ def samples_to_probs(sample_list: BSSamples) -> BSDistribution:
 
 def probs_to_sample_count(probs: BSDistribution, count: int = None, **kwargs) -> BSCount:
     count = _deduce_count(count, **kwargs)
+    if count < 1:
+        return BSCount()
     perturbed_dist = {state: max(prob + np.random.normal(scale=(prob * (1 - prob) / count) ** .5), 0)
                       for state, prob in probs.items()}
```

Comment:
It's just my ignorance but why do you add some randomness to the distributon. Will not it appear from the sampling automatically ?